### ISSUE TITLE
include name matching offline ITIS in addition to existing offline ITIS matching by id 

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/nomer#46]
as suggested by @seltmann .

**Comment by jhpoelen** [comment]
Implemented via https://github.com/globalbioticinteractions/nomer/releases/tag/0.2.4 . 

@seltmann please confirm that you are able to reproduce and close this issue: 

```
$ echo -e "\tHomo sapiens" | nomer append itis
using matcher [itis-taxon-id]
ITIS taxonomy already indexed at [/media/jorrit/data/nomer/itis/itis], no need to import.
	Homo sapiens	HAS_ACCEPTED_NAME	ITIS:180092	Homo sapiens	species		Animalia | Bilateria | Deuterostomia | Chordata | Vertebrata | Gnathostomata | Tetrapoda | Mammalia | Theria | Eutheria | Primates | Haplorrhini | Simiiformes | Hominoidea | Hominidae | Homininae | Homo | Homo sapiens	ITIS:202423 | ITIS:914154 | ITIS:914156 | ITIS:158852 | ITIS:331030 | ITIS:914179 | ITIS:914181 | ITIS:179913 | ITIS:179916 | ITIS:179925 | ITIS:180089 | ITIS:943773 | ITIS:943778 | ITIS:943782 | ITIS:180090 | ITIS:943805 | ITIS:180091 | ITIS:180092	kingdom | subkingdom | infrakingdom | phylum | subphylum | infraphylum | superclass | class | subclass | infraclass | order | suborder | infraorder | superfamily | family | subfamily | genus | species	http://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=180092	
```

and 

```
$ echo -e "\tArius felis" | nomer append itis
using matcher [itis-taxon-id]
ITIS taxonomy already indexed at [/media/jorrit/data/nomer/itis/itis], no need to import.
	Arius felis	SYNONYM_OF	ITIS:680665	Ariopsis felis	speciesAnimalia | Bilateria | Deuterostomia | Chordata | Vertebrata | Gnathostomata | Actinopterygii | Teleostei | Ostariophysi | Siluriformes | Ariidae | Ariopsis | Ariopsis felis	ITIS:202423 | ITIS:914154 | ITIS:914156 | ITIS:158852 | ITIS:331030 | ITIS:914179 | ITIS:161061 | ITIS:161105 | ITIS:162845 | ITIS:163992 | ITIS:164157 | ITIS:639019 | ITIS:680665	kingdom | subkingdom | infrakingdom | phylum | subphylum | infraphylum | superclass | class | superorder | order | family | genus | species	http://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=680665	
```
```

**Comment by jhpoelen** [comment]
available in 0.2.6 . please re-open if issues remain.
